docs/src/pages/components/pickers/pickers.md

```diff
@@ -24,20 +24,49 @@ packageName: '@material-ui/lab'
 
 ⚠️ Native input controls support by browsers [isn't perfect](https://caniuse.com/#feat=input-datetime).
 
-### Date picker
+Native date (`type="date"`), time (`type="time"`) and date&time (`type="datetime-local"`) pickers.
 
-A native datepicker example with `type="date"`.
+{{"demo": "pages/components/pickers/NativePickers.js"}}
```

Comment:
We lose the inline preview developers can easily copy & paste but we gain grouping to more easily compare the native pickers to our custom components. What's best? 

Did we consider reversing the grouping?

- date picker
  - react component
  - native
- time picker
  - react component
  - native
- etc.

Reply:
> We lose the inline preview developers can easily copy & paste

I don't believe this matters here.

Reply:
> I don't believe this matters here.

@eps1lon Yeah, if we assume that the primary intent is to showcase, to give an overview of what's available, I agree.